mkdocs.yml

```
@@ -89,6 +89,7 @@ nav:
   - 'Running Jobs':
     - running/index.md
     - 'slurm': running/slurm.md
+    - 'jobreport': running/jobreport.md
```

I think this can be
```suggestion
    - 'Job report': running/jobreport.md
```
to match the page heading.